bin/dist_build.sh

```diff
@@ -59,10 +58,8 @@ done
 
 echo "Building Tyk binaries"
 gox -tags 'goplugin' -osarch="linux/amd64 linux/386" -cgo
```

Comment:
If we're removing i386 wouldn't this cause the pipeline to fail when building the binaries?

Reply:
removed refs to 386